hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/placement/algorithms/TestContainerPlacementFactory.java

```diff
@@ -191,6 +194,13 @@ public List<DatanodeDetails> chooseDatanodes(
         validateContainerPlacement(List<DatanodeDetails> dns, int replicas) {
       return new ContainerPlacementStatusDefault(1, 1, 1);
     }
+
+    @Override
+    public Set<ContainerReplica> replicasToCopy(
+            Set<ContainerReplica> replicas, int expectedCountPerUniqueReplica,
+            int expectedUniqueGroups) {
+      return null;
```

Comment:
Probably best to return emptySet here rather than null.

Reply:
done